backend/handlers/auth.go

```diff
@@ -20,27 +20,33 @@ import (
 )
 
 const (
-	discordClientID    = "933567168912719923"
 	discordAPIEndpoint = "https://discord.com/api/v8"
 )
 
 var (
+	discordClientID string
```

Comment:
Good catch! While my Discord application should work without issues for the frontend, I realize that for the backend you need your own for dev purposes, since you require not only the ID but also the secret.